mobile/test/java/org/chromium/net/CronetUrlRequestContextTest.java

```diff
@@ -276,7 +269,6 @@ public void run() {
   @Test
   @SmallTest
   @Feature({"Cronet"})
-  @OnlyRunNativeCronet // JavaCronetEngine doesn't support throwing on repeat shutdown()
```

Comment:
What allows us to remove all these annotations?

Reply:
After removing java fallback impl, all the tests are running with native clients only now. There's no longer a need to specify.